### ISSUE TITLE
c-ares breaks binary builds

### ISSUE THREAD

**Issue by 0xxon** [zeek/zeek#2070]
The newly merged use of c-ares breaks our binary builds. Specifically, the error is:

```
[ 2830s] make[3]: *** No rule to make target 'cares-build/opt/zeek-nightly/lib/libcares.a', needed by 'src/zeek'.  Stop.
```

This seems to be caused by the interaction of a few configure options; specifically `--libdir` seems important (I did not check if any other options are necessary).

Calling configure with the following arguments will cause this error:

```
./configure --binary-package --enable-static-broker --enable-static-binpac --prefix=$HOME/test --libdir=$HOME/test/lib
```

**Comment by timwoj** [comment]
Gah, this is the ridiculous issue I was fighting on GitHub workflows as well. Is it only happening on debian builds?

**Comment by 0xxon** [comment]
No, it is happening on all builds.

**Comment by 0xxon** [comment]
In case you want to see the full output - you can just go to https://build.opensuse.org/package/show/security:zeek/zeek-nightly and click on any of the build errors. Currently all platforms that we are compiling for there are failing.

**Comment by 0xxon** [comment]
Fixed by #2071
